miscellaneous/di_i2c.py

```diff
@@ -373,8 +374,7 @@ def read_list(self, reg, len):
             outArr = []
         return self.transfer(outArr, len)
 
-
-import RPi.GPIO as GPIO
+import wiringpi as wp
```

Comment:
Can we import wiringpi as `wiringpi` and not as `wp`? `import wiringpi` vs. `import wiringpi as wp`

It would require corresponding changes below (find and replace `wp` with `wiringpi`).

Reply:
Done.